tests/test_scripts.py

```diff
@@ -52,13 +53,17 @@ def test_scripts(script, args):
     assert p.returncode == 0
 
 
-def test_bulk_process(tmpdir):
+def test_bulk_process(tmp_path):
```

Comment:
does this path automatically get deleted after the test is complete? i believe this is how `tmpdir` works; how about `tmp_path`?

Reply:
Will need to check.

Reply:
<img width="674" alt="image" src="https://github.com/mandiant/capa/assets/156560/673bdfc0-3a6d-4c23-b0aa-d4570227ee56">

looks like they are, so this is fine, nice change!